securitycenter/muteconfigv2/mute_config_test.go

```diff
@@ -0,0 +1,285 @@
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	securitycenter "cloud.google.com/go/securitycenter/apiv2"
+	"cloud.google.com/go/securitycenter/apiv2/securitycenterpb"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"github.com/golang/protobuf/ptypes"	
+	"github.com/google/uuid"
+)
+
+var fixture *muteconfigFixture
+
+func TestMain(m *testing.M) {
+	var err error
+	fixture, err = newMuteConfigFixture()
+	if err != nil {
+		log.Fatalf("failed to create fixture: %s", err)
+	}
+
+	exitCode := m.Run()
+
+	if err := fixture.Cleanup(); err != nil {
+		log.Fatalf("failed to cleanup resources: %s", err)
+	}
+	fixture.client.Close()
+	os.Exit(exitCode)
+}
+
+func createFinding(findingID string, category string, sourceName string, orgId string) (*securitycenterpb.Finding, error) {
+	ctx := context.Background()
+	client, err := securitycenter.NewClient(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("securitycenter.NewClient: %v", err)
+	}
+
+	eventTime, err := ptypes.TimestampProto(time.Now())
+	if err != nil {
+		return nil, fmt.Errorf("TimestampProto: %v", err)
+	}
+
+	req := &securitycenterpb.CreateFindingRequest{
+		Parent:    sourceName,
+		FindingId: findingID,
+		Finding: &securitycenterpb.Finding{
+			State: securitycenterpb.Finding_ACTIVE,
+			// Resource the finding is associated with.  This is an
+			// example any resource identifier can be used.
+			ResourceName: fmt.Sprintf("//cloudresourcemanager.googleapis.com/organizations/%s", orgId),
+			// A free-form category.
+			Category: category,
+			// The time associated with discovering the issue.
+			EventTime: eventTime,
+			Severity:  securitycenterpb.Finding_LOW,
+		},
+	}
+	return client.CreateFinding(ctx, req)
+}
+
+func createSource(w io.Writer, orgId string) error {
+	ctx := context.Background()
+	client, err := securitycenter.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("securitycenter.NewClient: %v", err)
+	}
+	source, err := client.CreateSource(ctx, &securitycenterpb.CreateSourceRequest{
+		Source: &securitycenterpb.Source{
+			DisplayName: "Customized Display Name",
+			Description: "A new custom source that does X",
+		},
+		Parent: fmt.Sprintf("organizations/%s", orgId),
+	})
+
+	if err != nil {
+		return fmt.Errorf("CreateSource: %v", err)
+	}
+	fmt.Fprintf(w, source.Name)
+	return nil
+}
+
+type muteconfigFixture struct {
+	client        *securitycenter.Client
+	orgId         string
+	projectId     string
+	parent        string
+	sourceName    string
+	finding1Name  string
+	finding2Name  string
+	muteConfigId1 string
+	muteConfigId2 string
+}
+
+// newMuteConfigFixture initializes variables in this file with resources to
+// use for testing.
+func newMuteConfigFixture() (*muteconfigFixture, error) {
+	var mc muteconfigFixture
+	var err error
+
+	mc.client, err = securitycenter.NewClient(context.Background())
+	if err != nil {
+		return nil, fmt.Errorf("failed to create security center client: %w", err)
+	}
+
+	orgId := os.Getenv("SCC_PROJECT_ORG_ID")
```

Comment:
Are these 2 env vars set in the test environment? Where do the values come from?

Reply:
I will set it in the terminal while testing in local, but they will be defined in the CI to fetch the values and use them.